src/Form/Form.js

```diff
@@ -65,67 +66,91 @@ const Form = props => {
 
   return (
     <form className={classes.form} onSubmit={e => handleSubmit(e)}>
-      <h3 className={classes.formTitle}>Add New Item</h3>
-      <input
-        className={classes.inputName}
-        type="text"
-        name="token"
-        placeholder="List Token"
-        value={token}
-        onChange={e => {
-          onEnterToken(e);
-        }}
-        required
-      />
-      <input
-        className={classes.inputName}
-        type="text"
-        name="name"
-        placeholder="Item Name"
-        value={name}
-        onChange={e => setName(e.target.value)}
-        required
-      />
       <div>
-        <label className={classes.inputLabel}>
+        <img
+          className={classes.groceryImg}
+          src={grocery}
+          alt="A green grocery bag full of groceries with a recycling symbol on the front. "
+        />
+      </div>
+
+      {/* <div className="input-field">
+        <input
+          type="text"
+          name="token"
+          value={token}
+          onChange={e => {
+            onEnterToken(e);
+          }}
+          required
+        />
+        <label for="token">List Token</label>
+      </div> */}
```

Comment:
Can you explain why this was commented out? Do we still need it?

Reply:
It was mentioned a bit in the PR Actions. We also asked whether anyone had an opinion on removing these in Slack, but didn't get a response, so I just commented them out in the short term. I'll just go ahead and remove them now.